node/chainSimulator/chainSimulator.go

```diff
@@ -626,7 +626,7 @@ func (s *simulator) computeTransactionsStatus(txsWithResult []*transactionWithRe
 
 		result, errGet := s.GetNodeHandler(destinationShardID).GetFacadeHandler().GetTransaction(resultTx.hexHash, true)
 		if errGet == nil && result.Status != transaction.TxStatusPending {
-			log.Info("############## transaction was executed ##############", "txHash", resultTx.hexHash)
+			log.Trace("############## transaction was executed ##############", "txHash", resultTx.hexHash)
```

Comment:
are these log level changes intended?

Reply:
There were informative logs, to see the hash of the transaction, changed to Trace because in the case of 10k transactions, we will see a lot of these messages in the log.